src/apify_client.ts

```diff
@@ -108,13 +110,9 @@ export class ApifyClient {
         });
     }
 
-    // TODO we don't have this endpoint yet
-    // /**
-    //  * @return {BuildCollectionClient}
-    //  */
-    // builds() {
-    //     return new BuildCollectionClient(this._options());
-    // }
+    builds(): BuildCollectionClient {
```

Comment:
Please add a reference to API docs into comments, the same as we had in the rest of the endpoints.